package.json

```diff
@@ -7,11 +7,13 @@
   "scripts": {
     "clean:caches": "rm -rf .eslintcache .stylelintcache node_modules/.cache",
     "dev": "vite",
-    "lint": "npm run lint:eslint && npm run lint:stylelint",
+    "fix:eslint": "npm run lint:eslint -- --config .eslintrc.fix.cjs --cache=false --fix",
+    "fix:stylelint": "npm run lint:stylelint -- --cache=false --config .stylelintrc.fix.js --fix",
```

Comment:
question: are these triggered in the pre-commit hook or just here for manual execution?

Reply:
They are for manual execution